regolith/helpers/l_contactshelper.py

```diff
@@ -108,19 +108,24 @@ def sout(self):
         if rc.date:
             date_list = []
             temp_dat = date_parser.parse(rc.date).date()
+            temp_dict = {
+                "begin_date": (temp_dat -
+                               dateutil.relativedelta.relativedelta(
+                                   months=int(
+                                       rc.range))).isoformat(),
+                "end_date": (temp_dat +
+                             dateutil.relativedelta.relativedelta(
+                                 months=int(
+                                     rc.range))).isoformat()}
             for contact in self.gtx["contacts"]:
-                if rc.date:
-                    temp_dict = {
-                        "begin_date": (temp_dat -
-                                       dateutil.relativedelta.relativedelta(
-                                           months=int(
-                                               rc.range))).isoformat(),
-                        "end_date": (temp_dat +
-                                     dateutil.relativedelta.relativedelta(
-                                         months=int(
-                                             rc.range))).isoformat()}
-                    if is_current(temp_dict, now=temp_dat):
-                        date_list.append(stringify(contact))
+                if contact.get('day'):
```

Comment:
this should be handled by ``get_dates()``.  If not, it may be a bug in get_dates().  Can you give more info about what is failing when you use get_dates?

Reply:
The error wasn't in using get_dates, it was that I was accidentally using rc.date as my "current date" to compare to rather than the actual date from the contact. I just uploaded a new commit that revises my code and uses get_dates().